{"pytest_version": "7.2.2", "$report_type": "SessionStart"}
{"nodeid": "tests42_64_5.py::test_list_of_primes", "location": ["tests42_64_5.py", 4, "test_list_of_primes"], "keywords": {"test_list_of_primes": 1, "tests42_64_5.py": 1, "Folder_64": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00013894500079913996, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "e83c6ca1af954ce692326652157ef882", "node": "<WorkerController gw0>"}
{"nodeid": "tests42_64_5.py::test_list_of_tens", "location": ["tests42_64_5.py", 15, "test_list_of_tens"], "keywords": {"test_list_of_tens": 1, "tests42_64_5.py": 1, "Folder_64": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00011712400009855628, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "e83c6ca1af954ce692326652157ef882", "node": "<WorkerController gw1>"}
{"nodeid": "tests42_64_5.py::test_list_of_primes", "location": ["tests42_64_5.py", 4, "test_list_of_primes"], "keywords": {"test_list_of_primes": 1, "tests42_64_5.py": 1, "Folder_64": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/home/shahin/Benchmark_Command/Q42/command_results_5/Folder_64/generated_answer.py", "lineno": 2, "message": "AttributeError: 'int' object has no attribute 'prime_factors'"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_primes():", "        for n in [2, 3, 5, 7, 11, 13, 17]:", "            list_of_primes = [n for _ in range((1 + 1) * 10)]", ">           assert prime_factors(list_of_primes) == {n}"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q42/command_results_5/Folder_64/tests42_64_5.py", "lineno": 8, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def prime_factors(nums):", ">       return set(nums[0].prime_factors())", "E       AttributeError: 'int' object has no attribute 'prime_factors'"], "reprfuncargs": {"args": [["nums", "[2, 2, 2, 2, 2, 2, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q42/command_results_5/Folder_64/generated_answer.py", "lineno": 2, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_primes():", "        for n in [2, 3, 5, 7, 11, 13, 17]:", "            list_of_primes = [n for _ in range((1 + 1) * 10)]", ">           assert prime_factors(list_of_primes) == {n}"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q42/command_results_5/Folder_64/tests42_64_5.py", "lineno": 8, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def prime_factors(nums):", ">       return set(nums[0].prime_factors())", "E       AttributeError: 'int' object has no attribute 'prime_factors'"], "reprfuncargs": {"args": [["nums", "[2, 2, 2, 2, 2, 2, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q42/command_results_5/Folder_64/generated_answer.py", "lineno": 2, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/home/shahin/Benchmark_Command/Q42/command_results_5/Folder_64/generated_answer.py", "lineno": 2, "message": "AttributeError: 'int' object has no attribute 'prime_factors'"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0001261709985556081, "extra": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "e83c6ca1af954ce692326652157ef882", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
